Dockerfile-node

```diff
@@ -31,8 +36,8 @@ COPY test /code/test
 ARG SUBSTRATE_CLI_GIT_COMMIT_HASH
 
 RUN \
-    /root/.cargo/bin/cargo build --profile production --bin subspace-node && \
-    mv target/production/subspace-node subspace-node && \
+    /root/.cargo/bin/cargo build --profile $PROFILE --bin subspace-node && \
+    mv target/$PROFILE/subspace-node subspace-node && \
```

Comment:
you mean `$PRODUCTION`?

Reply:
No, it is an environment variable with a profile